### PR TITLE
Add the github URL to PR annotations

### PR DESCRIPTION
**Pull request by PawelLipski**
Follow-up to #1091

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Release notes
 
-## New in git-machete 3.20.1
+## New in git-machete 3.21.0
 
+- add: `--urls` flag to `git machete github anno-prs` that also adds the URL of the PR to the annotation (contributed by @guyboltonking)
```
Comment (PawelLipski) [inline RELEASE_NOTES.md]: Also, config key to be added that would affect all subcommands of `github`

Comment (PawelLipski) [reply RELEASE_NOTES.md]: As for now, I'd rather leave the flag just in `anno-prs` to keep CLI bloat at bay

**Comment by PawelLipski** [comment]
@guyboltonking done, to be released soon 🚢

**Comment by guyboltonking** [comment]
> @guyboltonking done, to be released soon 🚢

Great; thanks for taking on the chores!

**Pull request merged**